### PR TITLE
Cli 34 Configure one flex project for many instances

### PR DESCRIPTION
**Pull request by DennyGD**

### CODE REVIEW

File: __snapshots__/flex-deploy.test.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ Options:
   -h, --help                Show help                                  [boolean]
   --serviceId               Service ID                                  [string]
 
-This project is not configured. Use 'kinvey flex init' to get started.
+This project is not configured. Use 'kinvey flex init' to get started or set additional options: serviceId.
```
[the remark this answers is not in the document]
Comment (DennyGD) [reply __snapshots__/flex-deploy.test.js]: How about 'Alternatively, use options: serviceId.'? 
We can use the same construct when it's more than one option.

File: __snapshots__/flex-list.test.js [hunk 2]
```diff
@@ -44,15 +46,18 @@ Options:
...
   --no-color                Disable colors                             [boolean]
   -h, --help                Show help                                  [boolean]
-  --domain                  Specify domain: 'app' or 'org'              [string]
+  --domain                  Specify domain: 'app' or 'org'
+                                                [string] [choices: "app", "org"]
   --id                      ID of app or org                            [string]
 
-Domain must be either 'app' or 'org'.
+Invalid values:
+  Argument: domain, Given: "invalidDomain", Choices: "app", "org"
```
[the remark this answers is not in the document]
Comment (DennyGD) [reply __snapshots__/flex-list.test.js]: It's yargs.

File: __snapshots__/flex-deploy.test.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (dokov) [reply __snapshots__/flex-deploy.test.js]: Yes, that works as well.

**Comment by dokov** [comment]
LGTM.

**Pull request merged**